content/200-concepts/100-components/02-prisma-client/060-full-text-search.mdx

```diff
@@ -82,14 +82,15 @@ Here's how the following queries would match that text:
 ## Adding Indexes
 
 To speed up your full-text queries, you should add an index to your database.
-Prisma Migrate currently doesn't support adding search indices, but you can
-easily add one yourself.
+
+While the Prisma Schema doesn't natively support GIN indices yet, you can easily
+define your own index in SQL:
 
 ```sql
-CREATE INDEX post_body_index ON posts USING GIN (body);
+CREATE INDEX post_body_index ON "Post" USING GIN (to_tsvector('english', "body"));
 ```
 
-You can continue using Prisma Migrate as you were before, it will ignore indexes
+And you can continue using Prisma Migrate as you were before, it will ignore indexes
 that it doesn't know about.
```

Comment:
What does that sentence mean really? Introspection looks at existing indexes, but ignores the ones it does not understand. Migrate only uses them as the basis for creating migrations, which could lead to indexes not in the Prisma schema to being _removed_  in the database.

Reply:
@Jan, do you mean that with MIgrate, if the search index is ignored, and thus not added to the schema, then it (of course) won't be added to the new database?

I was reading the current sentence as "Don't worry about manually adding a search index, because it will not impact your use of Migrate."

Reply:
This is probably kinda a nitpick: `it will ignore indexes that it doesn't know about.` You can only ignore things that exist. The indexes do not exist in the schema, so Prisma migrate can not ignore them.